ffi/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "msi_ffi"
+version.workspace = true
+authors.workspace = true
+description = "Basic FFI interface for reading Windows Installer (MSI) files, using the `msi` crate."
+repository.workspace = true
+keywords.workspace = true
+license.workspace = true
+readme.workspace = true
+edition.workspace = true
+
+[lib]
+name = "msi_ffi"
+crate-type = ["lib", "cdylib"]
+
+[dependencies]
+chrono = "0.4.37"
+msi = { path = "../" }
```

Comment:
I tried out running `cargo publish --dry-run -p msi_ffi` with your patch, and it gave me the error:

```
error: all dependencies must have a version specified when publishing.
dependency `msi` does not specify a version
Note: The published dependency will use the version from crates.io,
the `path` specification will be removed from the dependency declaration.
```

Reply:
I've added dependabot to automatically bump "msi" crate in the "msi_ffi" crate, so it automatically opens a PR in case you forget to update "msi_ffi" crate. However, you'll still need to publish the crate manually. That's some work, but considering the library is nearly finished, and there isn't active/frequent development (except bug fixes, etc...), I think it's okay.

Reply:
I've added dependabot to automatically bump "msi" crate in the "msi_ffi" crate, so it automatically opens a PR in case you forget to update "msi_ffi" crate. However, you'll still need to publish the crate manually. That's some work, but considering the library is nearly finished, and there isn't any active/frequent development (except bug fixes, etc...), I think it's okay.

Reply:
@mdsteele I've added dependabot to automatically bump "msi" crate in the "msi_ffi" crate, so it automatically opens a PR in case you forget to update "msi_ffi" crate. However, you'll still need to publish the crate manually. That's some work, but considering the library is nearly finished, and there isn't any active/frequent development (except bug fixes, etc...), I think it's okay.